images/base/Dockerfile.rootfs-ubuntu

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
+
+SHELL ["/bin/bash","-c"]
+
+RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
```

Comment:
please let this part to: https://github.com/ceph/ceph-container/pull/877

Reply:
are you saying it should be removed from this PR? not sure I follow how #877 adds it back here. Maybe I'm missing something.

Reply:
Yes it should be removed from that PR, for several reasons:

* It was picked from #877 
* the file structure is not good, we need to keep files in the ceph-releases directory

Basically, we don't need to re-organize the structure. I'd like to only keep the tool in this PR.
Thanks.